Thrive.csproj

```diff
@@ -19,6 +19,7 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="src\auto-evo\steps\ForceExtinction.cs" />
```

Comment:
Everything else in this file seems to be alphabetical except this line...

Reply:
Oh, indeed, it seems to be alphabetical, but only until a certain point (l.171 has a break in logic and it starts being messy afterwards). Still not a reason to fix it.